static/js/stream_popover.js

```diff
@@ -70,6 +76,13 @@ exports.hide_starred_messages_popover = function () {
     }
 };
 
+exports.hide_move_topic_to_stream_popover = function () {
```

Comment:
Should this get called in `popovers.hide_all()`?    `exports.hide_all_except_sidebars` I think needs a line to call this, unless I'm missing something.

Reply:
I am not sure, the popover kept open whenever it lost the focus, I saw the others popovers, and I found that, how could I hide the popover when it lost the focus??

Reply:
I would just call this from `exports.hide_all_except_sidebars`, which is the general function we call for hiding popovers in a range of situations where it makes sense to do so.